src/applications/vaos/services/appointment/transformers.js

```diff
@@ -156,14 +156,6 @@ export function transformVAOSAppointment(appt, useFeSourceOfTruthTelehealth) {
       reasonForAppointment,
       preferredTimesForPhoneCall: appt.preferredTimesForPhoneCall,
       requestVisitType: getTypeOfVisit(appt.kind),
-      type: {
```

Comment:
Turns out we're not even using this at all so it's better to remove it instead of renaming it.